x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.tsx

```diff
@@ -293,6 +296,15 @@ export function SuggestionPanel({
   }, [ExpressionRendererComponent]);
 
   const [lastSelectedSuggestion, setLastSelectedSuggestion] = useState<number>(-1);
+  // get user's selection from localStorage, this key defines if the suggestions panel will be hidden or not
+  const [hideSuggestions, setHideSuggestions] = useLocalStorage(
```

Comment:
@dej611 the biggest problem I see is this:
```
  if (suggestions.length === 0) {
    return null;
  }
```
When there are no suggestions then we don't display the suggestions panel. For example 
![image](https://user-images.githubusercontent.com/17003240/139674698-a19480c4-d6cd-481c-9459-7c9ae10a4c23.png)

if we don't compute them if the suggestions are hidden, how this will work? Can we use another variable? If yes, which?